web/app/scripts/social-costs/social-costs-directive.js

```diff
@@ -5,11 +5,35 @@
     function SocialCosts() {
         var module = {
             restrict: 'EA',
-            templateUrl: 'scripts/social-costs/social-costs.html',
+            templateUrl: function(elem, attrs) {
+                // Note that use of templateUrls is discouraged in most cases
```

Comment:
This way is fine, too.  It would mean you could get rid of the big comment and the attributes, but at the cost, as you say, of more duplicated directive boilerplate.